os/arch/arm/src/amebalite/amebalite_serial.c

```diff
@@ -1144,6 +1159,7 @@ void up_serialinit(void)
 	uart_register("/dev/ttyS1", &TTYS1_DEV);
 #endif
 #ifdef TTYS2_DEV
+	rtl8720e_up_setup_pin(&TTYS2_DEV);
```

Comment:
Hi 
Currently TTYS0 is currently linked to Loguart for Lite and Smart, which the pins will alreadt be Pull up at boot.
Since the loguart cannot be disabled and will cause problems.
I have added TTYS0 with a condition when uart console (loguart UART4) is not used.
TTYS1 has been added

Reply:
Could you leave comment on the code for what you said?
```
Currently TTYS0 is currently linked to Loguart for Lite and Smart, which the pins will be Pulled up at boot.
Since the loguart cannot be disabled and will cause problems,
I have added TTYS0 with a condition when uart console (loguart UART4) is not used.
```

Reply:
Later we could wonder why only S0 has the conditional. That's why I ask you for comment.

Reply:
I have just added the comment into the code